docs/docs/metrics.md

```diff
@@ -143,11 +119,8 @@ expect the names of these metrics to change if the affected code changes.
 
 ### Derived metrics (mean, p99, ...)
 
-Our metrics library calculates derived metrics like "mean" and "p99". Unfortunately, if reported to statsd, they
-do not only relate to the reporting interval but the whole live time of the app with some exponential weighting
-algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
+Our metrics library calculates derived metrics like "mean" and "p99." Unfortunately, if they are reported to statsd, they do not only relate to the reporting interval but the whole live time of the app with some exponential weighting algorithm. So try to build your dashboard around "counts" rather than "rates" where possible.
 
-### Statsd, derived statistics and metric names
+### Statsd, derived statistics, and metric names
 
-Statsd typically creates derived statistics (mean, P99) from what is reported. This might interact in a weird
-fashion with the derived statistics that our codahale metrics package reports.
+Statsd typically creates derived statistics (mean, p99) from what is reported. This might interact in a weird fashion with the derived statistics that our codahale metrics package reports.
```

Comment:
Marathon can report mean values. Statsd can compute mean values. The point here is: please make sure where you report/compute mean values.